2021-12-25 18:19:27,353: INFO: Strategy HitAndRun: Executing the initialize lifecycle method
2021-12-25 18:19:27,356: INFO: Current backtesting datetime 2020-12-29 00:00:00-05:00
2021-12-25 18:19:27,356: INFO: Strategy HitAndRun: Executing the before_starting_trading lifecycle method
2021-12-25 18:19:40,157: ERROR: 'NoneType' object has no attribute 'df'
2021-12-25 18:19:40,159: ERROR: Traceback (most recent call last):
  File "C:\Users\claudioa\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\lumibot\strategies\strategy_executor.py", line 476, in run
    self._run_trading_session()
  File "C:\Users\claudioa\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\lumibot\strategies\strategy_executor.py", line 422, in _run_trading_session
    self._before_starting_trading()
  File "C:\Users\claudioa\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\lumibot\strategies\strategy_executor.py", line 234, in func_output
    result = func_input(self, *args, **kwargs)
  File "C:\Users\claudioa\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\lumibot\strategies\strategy_executor.py", line 298, in _before_starting_trading
    self.strategy.before_starting_trading()
  File "c:\Trading\p2f\source\hit_and_run.py", line 81, in before_starting_trading
    expansion_breakout(self.bars, self.tickers)
  File "c:\Trading\p2f\source\hr_triggers.py", line 19, in expansion_breakout
    high = bars[ticker].df["high"].iloc[-1]
AttributeError: 'NoneType' object has no attribute 'df'

2021-12-25 18:19:40,170: INFO: Strategy HitAndRun: Executing the on_bot_crash event method
